applications/smart-distancing/ui/web_gui.py

```diff
@@ -65,6 +71,8 @@ def update(self, input_frame, nn_out, distances):
             line_thickness=3,
         )
 
+        birds_eye_window = vis_util.birds_eye_view(birds_eye_window, output_dict["detection_boxes"],
```

Comment:
add a `#TODO` to getting back to it with perspective consideration since the intention of birds eye view is changing the view point of an object.